src/assemble_workflow/assemble_args.py

```diff
@@ -8,15 +8,29 @@
 import logging
 from typing import IO
 
-
 class AssembleArgs:
     manifest: IO
     keep: bool
 
     def __init__(self) -> None:
+
+        def check_positive(value) -> int:
```

Comment:
You can add type for the input. `value: str`

Reply:
This does not allow me to block non-positive inputs.